File: PlaneRental/PlaneRental.Web/Binding/Models/ReservationModel.js

(function (cr) {
    var ReservationModel = function (reservationId, car, rentalDate, returnDate) {

        var self = this;

        self.ReservationId = ko.observable(reservationId);
        self.Plane = ko.observable(car);
        self.RentalDate = ko.observable(rentalDate);
        self.ReturnDate = ko.observable(returnDate);

        self.CancelRequest = ko.observable(false);
    }
    cr.ReservationModel = ReservationModel;
}(window.PlaneRental));

File: PlaneRental/PlaneRental.Web/Models/ReservationModel.cs
using PlaneRental.Client.Entities;
using System;
using System.Collections.Generic;
using System.Linq;

namespace PlaneRental.Web.Models
{
    public class ReservationModel
    {
        public int Plane { get; set; }
        public DateTime PickupDate { get; set; }
        public DateTime ReturnDate { get; set; }
    }
}
